src/types/BaseField.ts

```diff
@@ -11,12 +11,18 @@ export enum BaseFieldDataType {
 	BOOLEAN = 'boolean',
 }
 
+export enum BaseFieldScope {
+	PROPOSAL = 'proposal',
+	PRIMARY_ORGANIZATION = 'primary_organization',
```

Comment:
I named this `PRIMARY_ORGANIZATION` instead of just `ORGANIZATION` because I do anticipate us having base fields for other types of organization (e.g. partner organizations, co-applicants, etc).

An reasonable alternative approach would be to simply call this `ORGANIZATION` and introduce the concept of `primary` or `secondary` at the point that we actually implement those concepts.

For example, calling this `PRIMARY_ORGANIZATION` does imply that we are going to care about organization types at the base field level (as opposed to, for instance, at the application form field level which is where it might be more appropriate).

As I type this out, I'm realizing that I'd like to just do ORGANIZATION here.  We can always add to the enum / edit the enum later when we implement other org types if we decide base field is the correct place to do that.